pymongo/synchronous/command_cursor.py

```diff
@@ -382,6 +382,16 @@ def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
         self.close()
 
     def to_list(self) -> list[_DocumentType]:
+        """Converts the contents of this cursor to a list more efficiently than ``[doc async for doc in cursor]``.
```

Comment:
```suggestion
        """Converts the contents of this cursor to a list more efficiently than ``[doc for doc in cursor]``.
```

I don't think we need to supply that language here. That or we just say above.

Reply:
Sorry, which language?

Reply:
The `async` part. I see you addressed it earlier.